Source/WTF/wtf/win/RunLoopWin.cpp

```diff
@@ -74,6 +74,15 @@ void RunLoop::run()
     }
 }
 
+void RunLoop::iterate()
+{
+    MSG message;
```

Comment:
Instead of keeping RunLoop::cycle() buggy and adding a new function, could this instead be moved to RunLoop::cycle()?

This is really the implementation of RunLoop::cycle(), as expected by the generic code.

It is hard to program the normal code using RunLoop::cycle(), if on Windows port the cycle() means totally opposite of what it means in other platforms.